modules/silicon_design/main.tf

```diff
@@ -65,6 +71,10 @@ data "google_project" "existing_project" {
   project_id = var.project_name
 }
 
+data "google_project" "project" {
```

Comment:
maybe we could simply remove the count argument from `existing_project`?

Reply:
By default all RADLab modules creates a new project in which module's resources are spun up. **create_project** is the flag in _variables.tf_ which determines if you would like to create the new project or use existing project.

NOTE:   Line `count      = var.create_project ? 0 : 1` is specified to use the existing project if **create_project** is set to **false** where value of **project_name** in _variables.tf_ should be the _project id_ of the project where you would like to spin up the project.

Reply:
I see, my worried was that I wouldn't be able to access `project_id` if `create_project` was `1`, but that should be already coverred by the `local` `project` being set to `module.project_radlab_silicon_design.0` when this is the case.

Thanks for the clarification!